### PR TITLE
Openapi examples

### PR DESCRIPTION
**Pull request by jacksteroo**
Updated openapi examples to help users simulate api calls with examples

### CODE REVIEW

File: serve/app/dependencies/db_utils.py [hunk 1]
```diff
@@ -59,7 +61,13 @@ async def fetch_rows(
...
 
 async def get_handle_fid_for_addresses(
-    addresses: list[str],
+    addresses: Annotated[list[str], Body(
+        title="Addresses",
+        description="A list of addresses.",
+        examples=[
+        ["0x4114e33eb831858649ea3702e1c9a2db3f626446","0x8773442740c17c9d0f0b87022c722f9a136206ed"]
+        ]
+    )],
```
Comment (web123pro) [inline serve/app/dependencies/db_utils.py:70]: This is an internal function.  So, we should remove this change.

File: serve/app/dependencies/graph.py [hunk 2]
```diff
@@ -75,7 +75,13 @@ async def go_eigentrust(
...
 
 async def get_neighbors_scores(
-  fids: list[int],
+  fids: Annotated[list[int], Body(
+    title="Farcaster IDs",
+    description="A list of FIDs.",
+    examples=[
+      [1,2,3]
+    ]
+  )],
```
Comment (web123pro) [inline serve/app/dependencies/graph.py:84]: This is an internal function. So, we should revert all the changes in this file.

File: serve/app/routers/direct_router.py [hunk 3]
```diff
@@ -51,8 +71,18 @@ async def get_direct_following_for_handles(
...
+    description="A list of handles.",
+    examples=[
+      [
+        "farcaster.eth",
+        "varunsrin.eth",
+        "farcaster",
+        "v"
+      ]
+    ]
+  )],
```
Comment (web123pro) [inline serve/app/routers/direct_router.py:85]: This is an internal function. We should revert this.

File: serve/app/routers/graph_router.py [hunk 4]
```diff
@@ -133,8 +164,18 @@ async def get_neighbors_following_for_handles(
...
+    description="A list of handles.",
+    examples=[
+      [
+        "farcaster.eth",
+        "varunsrin.eth",
+        "farcaster",
+        "v"
+      ]
+    ]
+  )],
```
Comment (web123pro) [inline serve/app/routers/graph_router.py:178]: This is an internal api. We should revert this.

File: serve/app/routers/localtrust_router.py [hunk 5]
```diff
@@ -144,8 +169,18 @@ async def get_personalized_following_for_handles(
...
+    description="A list of handles.",
+    examples=[
+      [
+        "farcaster.eth",
+        "varunsrin.eth",
+        "farcaster",
+        "v"
+      ]
+    ]
+  )],
```
Comment (web123pro) [inline serve/app/routers/localtrust_router.py:183]: Internal function

**Comment by jacksteroo** [comment]
Resolved all of the above, plus any def that starts with underscore (_get...)

**Review by web123pro — APPROVED** [review]

**Pull request merged**